package.json

```
@@ -34,10 +34,10 @@
     "raw-body": "2.4.1"
   },
   "devDependencies": {
-    "@azure/cosmos": "3.10.2",
+    "@azure/cosmos": "3.17.3",
     "@types/jest": "^29.5.1",
     "@types/lru-cache": "5.1.0",
-    "@types/node": "^16.0.0",
+    "@types/node": "20",
```

Can we use the more common range specifier here?

```suggestion
    "@types/node": "^20.19.9",
```